src/assets/i18n/da.json

```diff
@@ -52,9 +52,10 @@
     "TOOLTIP_INFO": "Info - kan være af interesse for zonens administrator, men betyder ikke, at zonen indeholder et problem.",
     "TOOLTIP_NOTICE": "Notice - betyder noget, der skal være kendt af zonens administrator, men det er ikke nødvendigvis et problem.",
     "TOOLTIP_WARNING": "Warning - betyder noget, der under visse omstændigheder vil være et problem, men som sandsynligvis ikke vil blive bemærket af den almindelige bruger.",
+    "Test completed": null,
     "There is no delegation for the zone": "Der er ikke nogen delegation for zonen",
     "UNDELEGATED_PLURAL": "Ikke-delegerede",
     "UNDELEGATED_SINGULAR": "Ikke-delegeret",
     "What is an undelegated domain test?": "Hvad er en ikke-delegeret test?",
     "Zonemaster Backend is not available": "Zonemaster Backend er ikke tilgængelig"
-}
+}
```

Comment:
How are the files created? Why is newline missing on last line. It is easier to handle the file if it has.

Reply:
with `npm run i18n:extract`

Reply:
Can that be made to add a newline to the files?